### PR TITLE
[REve] RenderCore integration

### PR DESCRIPTION
**Pull request by osschar**
Picking, lights, render-queue/multi-pass rendering, support for most EveElements.

### CODE REVIEW

File: ui5/eve7/lib/EveScene.js [hunk 1]
```diff
@@ -40,9 +40,6 @@ sap.ui.define(['rootui5/eve7/lib/EveManager'], function(EveManager) {
 
       if (obj3d)
       {
-         // MT ??? why?, it can really be anything, even just container Object3D
-         obj3d._typename = "THREE.Mesh";
-
          // add reference to a streamed eve element to obj3d
```
Comment (linev) [inline ui5/eve7/lib/EveScene.js:43]: This `_typename` used by JSROOT.TGeoPainter. 
Normally JSROOT works with ROOT objects - all with `_typename` member.
Here I adding typename to the `three.js` object to simplify handling on JSROOT side. 
Please left it as it was before.

**Review by linev — CHANGES_REQUESTED** [review]

File: ui5/eve7/lib/EveScene.js (hunk 1 above)
Comment (osschar) [reply ui5/eve7/lib/EveScene.js:43]: OK, done ... thanks!

Comment (linev) [reply ui5/eve7/lib/EveScene.js:43]: But commit is still there - do you want to remove it?

**Comment by linev** [comment]
I rebase branch removing unnecessary commit.
Now it is ready for merge

**Review by linev — APPROVED** [review]

**Pull request merged**